src/main/java/com/mraof/minestuck/item/weapon/OnHitEffect.java

```diff
@@ -158,6 +160,117 @@
 		target.hurt(source, rng);
 	};
 	
+	OnHitEffect CAEGER_SWIPE = (stack, target, attacker) -> {
+		DamageSource source;
+		if(attacker instanceof Player player)
+			source = DamageSource.playerAttack(player);
+		else source = DamageSource.mobAttack(attacker);
+		if(stack.getItem() instanceof ICaegerMaker caegerMaker)
+		{
+			if(attacker instanceof Player player)
+			{
+				int coinAmount = caegerMaker.makeCaeger//this creates a certain amount of caegers for the player
+						(PlayerSavedData.getData(IdentifierHandler.encode(player), player.getLevel()));
+				for(int i=0; i<coinAmount; i++)
+				{
+					target.getLevel().addParticle(MSParticles.CaegerParticle.get()//spawns the particle
+							, target.getX(), target.getY(), target.getZ(),//gets the target location
+							(Math.random() - 0.5d) * 0.15d, 0.15d, (Math.random() - 0.5d) * 0.15d);//applies speed to particle in any given direction
+					
+				}
+			}
+		}
+	};
+	
+	OnHitEffect CAEGER_SWIPE_SCRATCH = (stack, target, attacker) -> {
+		DamageSource source;
+		if(attacker instanceof Player player)
+			source = DamageSource.playerAttack(player);
+		else source = DamageSource.mobAttack(attacker);
```

Comment:
how would i define the source

Reply:
Youve defined the source just fine. It looks like it would successfully apply damage the right way. However "source" is unused as theres no entity you are specifying to hurt. You could do something like the following in order to injure the entity being attacked: `target.hurt(source, amount);`

Reply:
well it's certainly not intended to do additional damage so if this is overriding the base damage of the weapon i don't think i need it here

Reply:
So caeger scratch is intended not to deal damage beyond the initial strike?